sdk/storage/azfile/directory/client_test.go

```diff
@@ -2188,3 +2188,99 @@ func (d *DirectoryRecordedTestsSuite) TestDirectoryRenameSourceTrailingDotAndOAu
 	_, err = newFileClient.GetProperties(context.Background(), nil)
 	_require.NoError(err)
 }
+
+func (d *DirectoryRecordedTestsSuite) TestListFileDirEncoded() {
+	_require := require.New(d.T())
+	testName := d.T().Name()
+
+	svcClient, err := testcommon.GetServiceClient(d.T(), testcommon.TestAccountDefault, nil)
+	_require.NoError(err)
+
+	shareClient := testcommon.CreateNewShare(context.Background(), _require, testcommon.GenerateShareName(testName), svcClient)
+	defer testcommon.DeleteShare(context.Background(), _require, shareClient)
+
+	dirName := "directory\uFFFF"
```

Comment:
Is this working or we need to use that slice to string conversion to create files with such characters in name?